### DISCUSSION TITLE
Siri and SimpliSafe plugin

### DISCUSSION THREAD

**Discussion by Githubnewbynewb** [homebridge-simplisafe3/homebridge-simplisafe3#254] [Q&A] [answered]
What are some best practices for using Siri and this plugin?  Siri often tells me I need to intervene on my phone. I’d like to instruct Siri to turn “home” on, etc. Thank you.

**Comment by shamoon** [comment, marked as answer]
Can you be a little more specific about "intervene"? HomeKit requires that certain devices like security systems, locks etc be confirmed on a device (or that the device is already unlocked). If thats what youre referring to theres not any direct way to fix it but you can use 'dummy switches' (homebridge-dummy) to workaround. For example, I have one setup when everyone leaves (geofencing) a dummy switch flips to "on" (and then switches back after 1 minute) and a separate automation that turns the alarm to "Away" when the switch turns on. This effectively circumvents the requirement.

Is that what you mean?

**Comment by Githubnewbynewb** [comment]
Thanks @shamoon, exactly what I was talking about.  I might try that.  I only have Hoobs, does Homebridge-dummy work with it?

**Reply by shamoon** [reply]
I believe so yea
